screens/Dashboard.js

```diff
@@ -23,10 +26,9 @@ class Dashboard extends Component {
   render() {
     const myData = this.state.StreaksData[0]
     return (
-      <view>
-        <Streak numOfStreaks = {this.state.StreaksData[0].data} />
-        <Streak numOfStreaks = {this.state.StreaksData[1].data} />
-      </view>
+      <View>
+        <CommonCompGroupUserList groupMembers={['Jake','Caroline','Patrick','Tony','Jipeng','Daniel','testuse','Onemore']}/>
```

Comment:
I think it is better if we save this in the state so that when we replace it with database call, it will be easier to refactor.